pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -821,6 +821,14 @@ type CommonPrometheusFields struct {
 	//
 	// +optional
 	TSDB *TSDBSpec `json:"tsdb,omitempty"`
+
+	// The name of the service name used by the underlying StatefulSet as its governing service.
```

Comment:
(nit) there are multiple statefulsets in case of sharding.
```suggestion
	// The name of the service name used by the underlying StatefulSet(s) as the governing service.
```